pinot-common/src/main/java/com/linkedin/pinot/common/data/Schema.java

```diff
@@ -288,6 +334,7 @@ public boolean validate(Logger ctxLogger) {
         switch (fieldType) {
           case DIMENSION:
           case TIME:
+          case DATE_TIME:
```

Comment:
can fieldType be a string/float/double?

Reply:
No, it's an enum consisting of only these values which are in the cases (DIMENSION, TIME, DATE_TIME, METRIC)

Reply:
I was referring to the switch below, that checks for different dataTypes.

Reply:
Yes, it can be string in case of more advanced simple date formats. And since it is an n units since epoch calculation, better to not restrict to whole numbers